Tests/ApolloTests/WebSocket/GraphqlTransportWsProtocolTests.swift

```diff
@@ -0,0 +1,276 @@
+import XCTest
+@testable import ApolloWebSocket
+import ApolloTestSupport
+import Nimble
+import Apollo
+import SubscriptionAPI
+
+class GraphqlTransportWsProtocolTests: XCTestCase {
+  private let asyncTimeout: DispatchTimeInterval = .seconds(3)
```

Comment:
A longer async wait interval for slower environments such as CI.

Reply:
These tests look great, but I'm really confused why CI would need a 3 second time out. If we are mocking the Socket and the Transport, it shouldn't need all that time. These are unit tests, not integration tests, right? 

If it's taking that long, I'm wondering if there is something going wrong with a locking thread or something?

Reply:
The longer timeout is my 'easy' fix attempt at resolving failed CI test. It's not that though, still working on resolving the failed tests.